_sass/blocks/explore/_regions-list.scss

```diff
@@ -86,12 +86,30 @@
   tr > *:first-child {
     padding-left: $base-padding;
     width: 80%;
+    &.narrow {
+      width: 50%;
+    }
   }
 
   tr > *:last-child {
 
   }
 
+  tr {
+    .variance,
+    .centered {
+      text-align: center;
+    }
+  }
+
+  tr > td > .reportee {
```

Comment:
Merge rule `tr > td > .reportee` with rule on line 98<br>Selector should have depth of applicability no greater than 3, but was 4